### DISCUSSION TITLE
How to use SPARQL Graph with other data graphs?

### DISCUSSION THREAD

**Discussion by Robert-Adelard** [ge-high-assurance/RACK#511] [Q&A]
By default, SPARQL Graph generates queries that search `http://rack001/data` and `http://rack001/model`, so how do I access data that has been uploaded to other data graphs, for example, `http://Turnstile/CounterApplication`?

**Comment by cuddihyge** [comment]
https://github.com/ge-semtk/semtk/wiki#a-quick-look

- Choose **load** off the **connection** menu.   
- Create a new connection (probably use the **Copy** button), and give your new connection a name
- Modify or add a new **Data** connection
- Hit **Submit**
